torch/package/package_exporter.py

```diff
@@ -255,7 +256,9 @@ def _get_dependencies(
             module_name if is_package else module_name.rsplit(".", maxsplit=1)[0]
         )
         try:
-            dep_pairs = find_files_source_depends_on(src, package_name)
+            dep_pairs = find_files_source_depends_on(
+                src, package_name, module_name, self._failed_dunder_imports
+            )
```

Comment:
Rather than pass in a list for `find_files_source_depends_on` to fill, consider returning the list from the function and then appending it to `self. _failed_dunder_imports`.